app/views/registration_transfers/success.html.erb

```diff
@@ -19,7 +19,7 @@
     </div>
 
     <div class="form-group">
-      <%= link_to t(".button"), continue_after_transfer_link(@registration), class: "button" %>
```

Comment:
Does this mean we can remove `continue_after_transfer_link` from RegistrationTransfersHelper?

Reply:
Yes! Good catch :)